cwms-data-api/src/main/java/cwms/cda/data/dto/LookupType.java

```diff
@@ -25,23 +25,23 @@
 package cwms.cda.data.dto;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
-import cwms.cda.api.errors.FieldException;
 import cwms.cda.formatters.Formats;
 import cwms.cda.formatters.annotations.FormattableWith;
 import cwms.cda.formatters.json.JsonV1;
 
-import java.util.Objects;
-
 @FormattableWith(contentType = Formats.JSON, formatter = JsonV1.class)
 @JsonDeserialize(builder = LookupType.Builder.class)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
-public final class LookupType implements CwmsDTOBase {
+public final class LookupType extends CwmsDTOBase {
```

Comment:
Updated LookupType and Property: https://github.com/USACE/cwms-data-api/pull/788
VerticalDatumInfo is in a similar boat, but unfortunately uses "office" instead of "office-id" so I'm leaving it for backwards compatibility.